src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+$bodyFont: 'Roboto';
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  height: 406px;
+  width: 200px;
 }
+
+@import './stars';
```

Comment:
Imports should be placed at the top of the file before any other styles are defined to ensure that the imported styles can be overridden if needed. Move the @import rules to the top of the file.